pom.xml

```diff
@@ -18,6 +18,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <checkstyle.skip>true</checkstyle.skip>
```

Comment:
I think this stops checkstyle from running on any child modules too. I pulled your change down, and when I do `mvn clean validate`, there is no `dropwizard-app/target/checkstyle-result.xml`. Commented it out, ran the same command, and it produced the report as expected.